### PR TITLE
feat: [PT-38] main action buttons module added + main page responsive…

### PR DESCRIPTION
**Pull request by EvgeniyKvasyuk**
… view

#### Description

Added actions button module for main page, responsive views added, some responsive fixes added

| Details                     | Description                                      |
| --------------------------- | ------------------------------------------------ |
| JIRA ticket                 | https://marathon-health.atlassian.net/browse/PT- |
| Design source               | N/A                                              |
| Related PRs                 | N/A                                              |
| Documentation update needed | No                                               |
| QA sign-off by              | N/A                                              |

## Changelog

- 

## Checklist

- [x] [Commit message convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [x] [Branch naming convention](https://kb.epam.com/display/MAHE/Code+Style) followed.
- [x] Performed self review & removed the debugging code (debugger, console log etc).
- [ ] Updated/Added tests.
- [ ] Added explanatory comment for tricky code explaining "why".
- [x] Tested on major browsers (Chrome, Firefox, Safari, Edge)
- [ ] Manual testing or integration testing done.
- [x] Added strings via locale

## Screenshots / Animated GIFs

Снимок экрана 2022-04-12 в 22 30 21
Снимок экрана 2022-04-12 в 22 30 02
Снимок экрана 2022-04-12 в 22 29 41

| Before | After |
| ------ | ----- |

### CODE REVIEW

File: src/modules/MainPageActions/MainPageActions.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+const cn = classNames.bind(styles);
+
+type Props = {
+  className?: string;
+};
+
+export const MainPageActions = ({ className }: Props) => {
+  const navigate = useNavigate();
+  const ACTIONS: ActionButtonProps[] = [
```
Comment (VladSolodjankin) [inline src/modules/MainPageActions/MainPageActions.tsx:23]: useMemo?

File: src/modules/MainPageActions/MainPageActions.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    } else {
+      rightColumn.push(<ActionButton key={i} className={cn('main-page-actions__button')} {...action} />);
+    }
+  });
+
+  return (
+    <Paper className={cn('main-page-actions', className)}>
+      <h3 className={cn('main-page-actions__heading')}>{locale.heading}</h3>
+      <div className={cn('main-page-actions__container')}>
+        {useMediaQuery(`(max-width: ${BREAK_POINTS.screenMobileMax})`) ? (
```
Comment (VladSolodjankin) [inline src/modules/MainPageActions/MainPageActions.tsx:69]: let's move it to constant useMediaQuery(`(max-width: ${BREAK_POINTS.screenMobileMax})`) to make render more clear and readable.

Comment (VladSolodjankin) [inline src/modules/MainPageActions/MainPageActions.tsx:69]: let's move it to constant useMediaQuery(`(max-width: ${BREAK_POINTS.screenMobileMax})`) to make render more clear and readable.

File: src/modules/MainPageActions/MainPageActions.module.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    max-width: calc(25% - 1rem);
+
+    @media (max-width: $screenMobileMax) {
+      width: 100%;
+      max-width: 100%;
+      margin-bottom: 1rem;
+    }
+  }
+
+  &__left-column {
```
Comment (VladSolodjankin) [inline src/modules/MainPageActions/MainPageActions.module.css:31]: we can create mixin for column styles

Comment (VladSolodjankin) [inline src/modules/MainPageActions/MainPageActions.module.css:31]: we can create mixin for column styles

Comment (EvgeniyKvasyuk) [reply src/modules/MainPageActions/MainPageActions.module.css:31]: I think  one __colum is enough. Will change.

**Review by VladSolodjankin — APPROVED** [review]

**Pull request merged**